src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -112,6 +112,112 @@
      f'--parent_run_id=' + dbutils.widgets.get('parent_run_id'))
 """
 
+EXPORT_TO_EXCEL_NOTEBOOK = """
```

Comment:
Why do we need this notebook as you introduced a command line export already?

Reply:
@JCZuurmond 
This notebook is intended for Clients where They don't have internet connectivity. They only have connectivity to internet via jump host, hence the hybrid approach is needed.